cinder/volume/manager.py

```diff
@@ -5057,14 +5175,20 @@ def _connection_terminate(self,
             err_msg = (_('Terminate volume connection failed: %(err)s')
                        % {'err': err})
             LOG.exception(err_msg, resource=volume)
+            action_track.track(
+                context, action_track.ACTION_VOLUME_DETACH,
+                volume, err_msg, failed=True
+            )
             raise exception.VolumeBackendAPIException(data=err_msg)
-        LOG.info("Terminate volume connection completed successfully.",
-                 resource=volume)
         # NOTE(jdg): Return True/False if there are other outstanding
         # attachments that share this connection.  If True should signify
         # caller to preserve the actual host connection (work should be
         # done in the brick connector as it has the knowledge of what's
         # going on here.
```

Comment:
Same here, should be right above the `return`.

Reply:
done